base/float.jl

```diff
@@ -720,9 +720,21 @@ exponent_one(::Type{Float16}) =     0x3c00
 exponent_half(::Type{Float16}) =    0x3800
 significand_mask(::Type{Float16}) = 0x03ff
 
+# integer size of float
+fpintsize(::Type{Float64}) = UInt64
+fpintsize(::Type{Float32}) = UInt32
+fpintsize(::Type{Float16}) = UInt16
```

Comment:
`fpintsize` sounds like it should return a number (`64` or `8`)... maybe `fpint` or `fpinttype`?